config/owasp/suppressions.xml

```diff
@@ -85,4 +85,11 @@
         <packageUrl regex="true">^pkg:maven/org\.postgresql/postgresql@.*$</packageUrl>
         <cve>CVE-2022-21724</cve>
     </suppress>
+    <suppress until="2022-05-01">
```

Comment:
As this is affecting the master pipeline, this should put in a separate PR

Reply:
will remove